tests/__mocks__/parser/resolve.mock.js

```diff
@@ -0,0 +1,32 @@
+const parser = jest.genMockFromModule('../../../lib/parser/index.js');
+const path = require('path');
+
+function mockPromise(value) {
+	return (value || {}).then ? value : {
+		then: function (callback) {
+			return mockPromise(callback(value));
+		}
+	};
+}
+function spawnChild(pkg) {}
```

Comment:
Here the linter also complains about:
* 'pkg' is defined but never used 
* 'result' is defined but never used